src/lib/functions/runtimes/js/index.ts

```diff
@@ -104,6 +106,13 @@ export const invokeFunctionDirectly = async ({ context, event, func, timeout })
   const lambdaPath = func.buildData?.buildPath ?? func.mainFile
   const result = await lambdaLocal.execute({
     clientContext: JSON.stringify(context),
+    environment: {
+      // We've set the Blobs context on the parent process, which means it will
+      // be available to the Lambda. This would be inconsistent with production
+      // where only V2 functions get the context injected. To fix it, unset the
+      // context variable before invoking the function.
+      [BLOBS_CONTEXT_VARIABLE]: undefined,
```

Comment:
Looking at the implementation of this, it seems like it mutates `process.env`: 

https://github.com/ashiina/lambda-local/blob/95fb3dfd18ec51014d3cc5d9f405e21a5d784a50/src/lambdalocal.ts#L250-L252

This leads to weird side-effects, like the env variable vanishing upon the first V1 Function execution. In this specific instance, it's probably alright because the framework child process was started earlier, and because V2 Functions don't rely on this env variable. But might be worth adding a comment.